src/Apps/ArtQuiz/Components/ArtQuizCard.tsx

```diff
@@ -113,13 +113,20 @@ const reducer = <T,>(state: State<T>, action: Action) => {
 }
 
 interface UseArtQuizCards<T> {
+  //** An array of card objects */
   cards: T[]
+  findIndexCallback?: (card: T) => boolean
```

Comment:
Realize I should switch this name to be more descriptive in the context of this component. Will also add useful pragma.

Reply:
What about just accepting an `initialIndex` prop and calculating this outside in the component that calls the hook?

Reply:
_Too easy @dzucconi_. 😝

Reply:
:) but yeah there's precedent for that as a prop. [Our older carousel and swiper for instance.](https://github.com/search?q=repo%3Aartsy%2Fpalette%20initialIndex&type=code)

Reply:
No, I know... I making fun of myself. I fixed it.